### ISSUE TITLE
Ranks(and stuff) in name tags

### ISSUE THREAD

**Issue by NoXZos** [svenar-nl/PowerRanks#10]
**I'm not a dummy, so I've checked these**
- [ X ] I am not using any other permission management plugin
- [ X ] I use the latest available version of PowerRanks
- [ X ] I am a human

Placeholders are a pain to use.
NameTagEdit is pretty confusing.

Solution?
A built in feature for ranks prefixes and suffixes in name tags.

Alternative???
Idk man automatically put them in?

**Additional context**
Basically something like this but less painful to use
![maxresdefault](https://user-images.githubusercontent.com/57535500/96366154-6600ac80-1178-11eb-8ae3-d811a3e4cca4.jpg)

**Comment by svenar-nl** [comment]
This has been requested numerous time now and I am working on integrating the NameTagEdit API to handle this.
